src/FilterBundle/DependencyInjection/BestItCommercetoolsFilterExtension.php

```diff
@@ -32,6 +32,9 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setAlias('best_it_commercetools_filter.request.client', $config['client_id']);
         $container->setAlias('best_it_commercetools_filter.generator.url', $config['url_generator_id']);
 
+        // Set parameters
+        $container->setParameter('best_it_commercetools_filter.cache_life_time', $config['cache_life_time']);
```

Comment:
imho: `->defaultValue(86400)` should be changed because i would expect a common project usage which is a cache behavior because of the nature of attribute labels in commercetools query result. also adding a tagged 5.x is easy ;)